File: eCinema/eCinema.Model/Cinema.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCinema.Model
{
    public class Cinema
    {
        public int CinemaId { get; set; }
        public string Naziv { get; set; }
        public string Adresa { get; set; }
        public string Webstranica { get; set; }
        public string Email { get; set; }
        public string BrTelefona { get; set; }
        public string? Logo { get; set; }
        public bool Aktivan { get; set; }
    }
}

File: ecinema_mobile/lib/models/cinema.dart
import 'package:json_annotation/json_annotation.dart';

part 'cinema.g.dart';

@JsonSerializable()
class Cinema {
  int cinemaId;
  String naziv;
  String adresa;
  String webstranica;
  String email;
  String brTelefona;
  String? logo;
  bool aktivan;

  Cinema(
      {required this.cinemaId,
      required this.naziv,
      required this.adresa,
      required this.webstranica,
      required this.email,
      required this.brTelefona,
      this.logo,
      required this.aktivan});

  factory Cinema.fromJson(Map<String, dynamic> json) => _$CinemaFromJson(json);
  Map<String, dynamic> toJson() => _$CinemaToJson(this);
}
